data/governance.yml

```
@@ -0,0 +1,522 @@
+    github: dra27
+    role: Maintainer
+  - name: Enguerrand
+    github: Engil
+    role: Maintainer
+  - name: Florian Angeletti
+    github: Octachron
+    role: Maintainer
+  - name: "Fr\xE9d\xE9ric Bour"
+    github: let-def
+    role: Maintainer
+  - name: Gabriel Scherer
+    github: gasche
+    role: Maintainer
+  - name: Jacques Garrigue
+    github: garrigue
+    role: Maintainer
+  - name: Jacques-Henri Jourdan
+    github: jhjourdan
+    role: Maintainer
+  - name: Jeremy Yallop
+    github: yallop
+    role: Maintainer
+  - name: KC Sivaramakrishnan
+    github: kayceesrk
+    role: Maintainer
+  - name: Leo White
+    github: lpw25
+    role: Maintainer
+  - name: Luc Maranget
+    github: maranget
+    role: Maintainer
+  - name: Mark Shinwell
+    github: mshinwell
+    role: Maintainer
+  - name: "Nicol\xE1s Ojeda B\xE4r"
+    github: nojb
+    role: Maintainer
+  - name: Pierre Chambart
+    github: chambart
+    role: Maintainer
+  - name: Sadiq Jaffer
+    github: sadiqj
+    role: Maintainer
+  - name: Stephen Dolan
+    github: stedolan
+    role: Maintainer
+  - name: "S\xE9bastien Hinderer"
+    github: shindere
+    role: Maintainer
+  - name: Thomas Refis
+    github: trefis
+    role: Maintainer
+  - name: Vincent Laviron
+    github: lthls
+    role: Maintainer
+- id: platform
+  name: Platform
+  description: The Platform team is responsible for the development and maintenance
+    of the OCaml Platform, the official OCaml developer toolchain.
+  contacts:
+  - name: Mailing List
+    link: https://lists.ocaml.org/listinfo/platform
+    kind: email
+  members:
+  - name: Anil Madhavapeddy
+    github: avsm
+    role: Owner
+  subteams:
+  - id: dune-dev
+    name: Dune
+    description: A composable build system for OCaml
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml/dune
+      kind: github
+    members:
+    - name: Rudi Grinberg
+      github: rgrinberg
+      role: Maintainer
+    - name: Ali Caglayan
+      github: Alizter
+      role: Maintainer
+    - name: Nicolás Ojeda Bär
+      github: nojb
+      role: Maintainer
+    - name: Marek Kubica
+      github: Leonidas-from-XIV
+      role: Maintainer
+    - name: Etienne Millon
+      github: emillon
+      role: Maintainer
+    - name: Stephen Sherratt
+      github: gridbugs
+      role: Maintainer
+    - name: Antonio Nuno Monteiro
+      github: anmonteiro
+      role: Maintainer
+    - name: Javier Chávarri
+      github: jchavarri
+      role: Maintainer
+    - name: Andrey Mokhov
+      github: snowleopard
+      role: Maintainer
+    - name: Emilio Jesús Gallego Arias
+      github: ejgallego
+      role: Maintainer
+    - name: Françoit Botot
+      github: bobot
+      role: Maintainer
+  - id: opam-dev
+    name: Opam
+    description: opam is a source-based package manager. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow.
```

```suggestion
    description: Opam is a source-based package manager. It supports multiple simultaneous compiler installations, flexible package constraints, and a Git-friendly development workflow.
```

I know formatting opam is very strange and doesn’t really adhere to normal grammar rules!